sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 5.2.0-beta.3 (Unreleased)
+- Default scheme to 'sb://' if not scheme is set in 'Endpoint'. 
```

Comment:
"no scheme". This has no impact to any users. So is this considered a "change log"?

Reply:
User do not have to change any API. But surfacing what we change, will give visibility to user when they are looking through changelog related to scheme.